Mathlib/Analysis/Calculus/IteratedDeriv/Lemmas.lean

```diff
@@ -91,6 +95,35 @@ theorem iteratedDerivWithin_sub (hf : ContDiffOn 𝕜 n f s) (hg : ContDiffOn 
 
 end
 
+lemma iteratedDeriv_add (hf : ContDiff 𝕜 n f) (hg : ContDiff 𝕜 n g) :
+    iteratedDeriv n (f + g) x = iteratedDeriv n f x + iteratedDeriv n g x := by
+  simpa only [iteratedDerivWithin_univ] using
+    iteratedDerivWithin_add (Set.mem_univ _) uniqueDiffOn_univ
+      (contDiffOn_univ.mpr hf) (contDiffOn_univ.mpr hg)
+
+theorem iteratedDeriv_const_add (hn : 0 < n) (c : F) :
+    iteratedDeriv n (fun z => c + f z) x = iteratedDeriv n f x := by
+  simpa only [iteratedDerivWithin_univ] using
+    iteratedDerivWithin_const_add (Set.mem_univ _) uniqueDiffOn_univ hn c
+
+theorem iteratedDeriv_const_neg (hn : 0 < n) (c : F) :
+    iteratedDeriv n (fun z => c - f z) x = iteratedDeriv n (-f) x := by
```

Comment:
This is a `sub` theorem not a `neg` theorem

Reply:
So it is. Fixed